src/Cli.Tests/ValidateConfigTests.cs

```diff
@@ -114,6 +114,37 @@ public void TestValidateWithEmptyConfig()
         }
     }
 
+    /// <summary>
+    /// This Test is used to verify that the validate command is able to catch invalid values for the depth-limit property.
+    /// </summary>
+    [DataTestMethod]
+    [DataRow("null", true, false, DisplayName = "Invalid Value: 'null'")]
+    [DataRow("20", true, true, DisplayName = "Valid Value: 20")]
+    [DataRow(-1, false, false, DisplayName = "Invalid Value: -1")]
+    [DataRow(0, false, false, DisplayName = "Invalid Value: 0")]
+    [DataRow(-2, false, false, DisplayName = "Invalid Value: -2")]
+    [DataRow(2147483648, false, false, DisplayName = "Invalid Value: 2147483648")]
+    [DataRow("seven", true, false, DisplayName = "Invalid Value: 'seven'")]
+    public void TestValidateConfigFailsWithInvalidGraphQLDepthLimit(object? depthLimit, bool isStringValue, bool isSuccessExpected)
```

Comment:
The data type on the runtime options property enforces that this value must be an integer.. i think it would be better to exclude the last test case of "seven" and make the test method paramter input data type an `int?`